src/shared/header/header.component.html

```diff
@@ -65,69 +65,86 @@ <h2 #headerTitle class="header-title heading pull-left"></h2>
+                          role="menu"
+                          aria-labelledby="single-button"
+                          class="dropdown-container">
+                        <li class="dropdown-item"
+                            role="menuitem"
+                            (click)="timeUnitFilterSelect('DAY')">
+                          <span class="dropdown-item">Day</span>
+                        </li>
+                        <li class="dropdown-item"
+                            role="menuitem"
+                            (click)="timeUnitFilterSelect('WEEK')">
+                          <span class="dropdown-item">Week</span>
+                        </li>
+                        <li class="dropdown-item"
+                            role="menuitem"
+                            (click)="timeUnitFilterSelect('MONTH')">
+                          <span class="dropdown-item">Month</span>
+                        </li>
+                        <li class="dropdown-item"
+                            role="menuitem"
+                            (click)="timeUnitFilterSelect('YEAR')">
+                          <span class="dropdown-item">Year</span>
+                        </li>
+                      </ul>
+                    </div>
 
-                  <div dropdown class="control-section">
-                    <button type="button"
-                            class="dropdown-button"
-                            dropdownToggle
-                            [disabled]="disabled">
-                      <span class="dropdown-selected">{{ timeUnitTemp?.name }}</span>
-                      <img src="assets/img/lang-selector-arrow.svg" alt="">
-                    </button>
-
-                    <ul dropdownMenu
-                        role="menu"
-                        aria-labelledby="single-button"
-                        class="dropdown-container">
-                      <li class="dropdown-item"
-                          role="menuitem"
-                          (click)="timeUnitFilterSelect('DAY')">
-                        <span class="dropdown-item">Day</span>
-                      </li>
-                      <li class="dropdown-item"
-                          role="menuitem"
-                          (click)="timeUnitFilterSelect('WEEK')">
-                        <span class="dropdown-item">Week</span>
-                      </li>
-                      <li class="dropdown-item"
-                          role="menuitem"
-                          (click)="timeUnitFilterSelect('MONTH')">
-                        <span class="dropdown-item">Month</span>
-                      </li>
-                      <li class="dropdown-item"
-                          role="menuitem"
-                          (click)="timeUnitFilterSelect('YEAR')">
-                        <span class="dropdown-item">Year</span>
-                      </li>
-                    </ul>
                   </div>
 
+
```

Comment:
@Miffael empty lines? for what?

Reply:
separate structure and logic (js work) components